README.md

```diff
@@ -4,13 +4,27 @@ This contains the source for QuPath's documentation, hosted at https://qupath.re
 
 ## Building locally
 
-To build this locally, you should first install (possibly in a [venv](https://docs.python.org/3/library/venv.html)):
-- `sphinx-build`
-- `sphinx_rtd_theme`
-- `myst_parser`
-- `readthedocs-sphinx-search`
+To build this locally, create a virtual environment, eg:
 
-As well as the command line tool `Make` (e.g., [GNU Make](https://www.gnu.org/software/make/)).
+```bash
+python -m venv ./venv
+. ./venv/bin/activate
+```
+
+or using conda/mamba:
+
+```bash
+conda env create -n qupath-docs
+conda activate qupath-docs
+```
+
+Then install the requirements for this repo:
+
+```bash
+pip install -r requirements.txt
+```
+
+You'll also need the command line tool `Make` (e.g., [GNU Make](https://www.gnu.org/software/make/)).
```

Comment:
Does this need to be installed separately on Windows?
I think macOS/Linux should generally have it - right?

Reply:
I would have to check later, I suspect you can install it with conda so it'd be easiest to point people that way

Yeah afaik mac and linux will always have some form of make